FilmFestivalLoader/IFFR/parse_iffr_html.py

```diff
@@ -559,24 +559,32 @@ def pr_debug(s):
             if FilmInfoPageParser.debugging:
                 Globals.debug_recorder.add('AC ' + s)
 
+        def str_extras(extras):
+            return ' [' + ', '.join([' '.join([str(e[0]), e[1].name]) for e in extras]) + ']'
+
+        def pr_debug_dict(d):
+            str_dict = '\n'.join([str(mf) + str_extras(extras) for (mf, extras) in d.items()])
+            pr_debug(f'Main films and extras:\n{str_dict}')
+
         def short_str(film_id):
             return iffr_data.get_film_from_id(film_id).short_str()
 
-        pr_debug(f'Main films and extras: {Globals.extras_by_main}')
+        pr_debug_dict(Globals.extras_by_main)
```

Comment:
Please explain why so much debug code is necessary.

Reply:
The previous debug code (statement) delivered completely unreadable text.
Since we still have a user story pending concerning directly combined films, I supposed making the text readable would be useful. For testing now, and for working on that user story.